libavfilter/vf_libvmaf.c

```diff
@@ -77,6 +97,7 @@ static const AVOption libvmaf_options[] = {
     {"n_subsample", "Set interval for frame subsampling used when computing vmaf.",     OFFSET(n_subsample), AV_OPT_TYPE_INT, {.i64=1}, 1, UINT_MAX, FLAGS},
     {"model",  "Set the model to be used for computing vmaf.",                          OFFSET(model_cfg), AV_OPT_TYPE_STRING, {.str="version=vmaf_v0.6.1"}, 0, 1, FLAGS},
     {"feature",  "Set the feature to be used for computing vmaf.",                      OFFSET(feature_cfg), AV_OPT_TYPE_STRING, {.str=NULL}, 0, 1, FLAGS},
+    {"metadata_handler",  "Set the feature to be propagated as metadata.",              OFFSET(metadata_feature_cfg), AV_OPT_TYPE_STRING, {.str="name=vmaf"}, 0, 1, FLAGS},
```

Comment:
I think it would be better to have a bool option to enable metadata writing. If you do it this way, only one feature can be written as metadata. When the option is set, you can register metadata callbacks for all registered features and models.

Reply:
individual feature names as well as model names should both be available during init.

Reply:
Actually users can give metadata handlers like this `libvmaf='n_threads=30:feature=name=psnr|name=ciede:metadata_handler=name=vmaf|name=psnr_y|name=ciede2000'` But obviously this makes hard to count metadata handlers, if someone enters wrong handler it still register that metadata handler.

Reply:
> individual feature names as well as model names should both be available during init.

I couldn't find a list of available feature names/models. Can you provide a link for me please?
Thanks